src/FcarrascosaSlider.js

```diff
@@ -50,9 +50,21 @@ export default class FcarrascosaSlider extends LitElement {
 
   constructor() {
     super();
+    this.onSwipeStart = this.onSwipeStart.bind(this);
+    this.onSwipeEnd = this.onSwipeEnd.bind(this);
+    this.onDrag = this.onDrag.bind(this);
+
     this.currentSlide = 0;
     this.latestSelectedSlide = null;
     this.totalAmountOfSlides = this.querySelectorAll('fcarrascosa-slider-slide').length;
```

Comment:
maybe you can use MutationObserver to get all slides into.

Reply:
Ill have a look into this, but this will be in another feature/revision, as it could be applied to other partsof this component.